VERSION

```diff
@@ -1 +1 @@
-1.17.0
+1.15.0.post2
```

Comment:
Why did this change?

Reply:
Could you read the "Version changes?" note in the PR description and let mw know what seems wrong or confusing about it?

Reply:
My bad, I didn't realize this change is being backported to prior releases.

Reply:
no prob